libDCM/libDCM.c

```diff
@@ -180,10 +180,11 @@ void udb_heartbeat_callback(void)
 void dcm_calibrate(void)
 {
 	// Don't allow re/calibrating before the initial calibration period has finished
-	if (dcm_flags._.calib_finished)
-	{
+	// This FIX ONLY Experimental for ROllPitchYaw, until the calling of dcm_calibrate is fully understood.

```

Comment:
this should be done in a separate commit which can later be reverted. Do not commit comment code